about.html

```diff
@@ -0,0 +1,311 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;0,900;1,400&display=swap" rel="stylesheet">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+        crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/main.css">
+
+    <title>Capstone</title>
+</head>
+
+<body>
+    <div class="container-fluid">
+        <!--upper navigation-->
+
+
+        <div class="row top_navigation">
+            <div class="col-7"></div>
+            <div class="col-3">
+                <ul>
+                    <li>
+                        <img src="images/facebook.png" alt="facebook">
+                    </li>
+                    <li>
+                        <img src="images/twitter.png" alt="twitter">
+                    </li>
+                    <li>English</li>
+                    <li>My page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="col-2"></div>
+        </div>
+
+        <!--Lower navigation-->
+        <div class="row lower_navigation">
+            <div class="col-1"></div>
+            <div class="col-2 about_logo">
+                <img src="images/women.png" alt="">
+            </div>
+            <div class="col-1"></div>
+            <div class="col-8">
+                <ul>
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="##">Program</a>
+                    </li>
+                    <li>
+                        <a href="##">Join</a>
+                    </li>
+                    <li>
+                        <a href="##">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="##">News</a>
+                    </li>
+                    <li>
+                        <a href="##">Follow us</a>
+                    </li>
+                </ul>
+            </div>
+
+        </div>
+
+
+
+
+        <!--main section-->
+        <div class="container-fluid main-section" >
```

Comment:
- [ Required ] Please consider changing this tag to the HTML5 semantic `main` tag, the `main` tag is usually used to wrap the main content of a page, instead of using generic classes with the similar meaning. 
```HTML
<main class="container-fluid main-section">
</main>
```